front/app/components/admin/ContentBuilder/typings.ts

```diff
@@ -6,8 +6,10 @@ export type ContentBuilderErrors = Record<
   { hasError: boolean; selectedLocale?: Locale }
 >;
 
+export type CraftJson = Record<string, SerializedNode>;
```

Comment:
Don't you need to add this type to `IProjectDescriptionBuilderData` in `app/modules/commercial/project_description_builder/api/types.ts`?

Reply:
It is added already in the attributes. All necessary changes have been made, I think.